spackle-android-lib/src/main/kotlin/co/electriccoin/zcash/spackle/Twig.kt

```diff
@@ -0,0 +1,81 @@
+package co.electriccoin.zcash.spackle
+
+import android.content.Context
+import android.util.Log
+import co.electriccoin.zcash.spackle.process.ProcessNameCompat
+import java.util.Locale
+
+/**
+ * A twig is a tiny log. These logs are intended for development rather than for high performance
+ * or usage in production.
+ */
+object Twig {
```

Comment:
Nice to have this in the project!

Reply:
I need to add a ProGuard configuration to strip logs during release builds.